package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.7.2",
   "private": true,
   "engines": {
-    "node": "11.15.0"
+    "node": "12.0.0"
```

Comment:
@zacck have you tried to build our docker image after this change? we are fixed in `11.15.0` mainly because the `docker build` cmd fails with other node versions

Reply:
I checked the builds but not in docker ... let me see how it goes